src/Chainweb/Version/FastDevelopment.hs

```diff
@@ -0,0 +1,58 @@
+{-# language LambdaCase #-}
+{-# language NumericUnderscores #-}
+{-# language OverloadedStrings #-}
+{-# language PatternSynonyms #-}
+{-# language QuasiQuotes #-}
+{-# language ViewPatterns #-}
+
+module Chainweb.Version.FastDevelopment(fastDevnet, pattern FastDevelopment) where
+
+import Chainweb.BlockCreationTime
+import Chainweb.ChainId
+import Chainweb.Difficulty
+import Chainweb.Graph
+import Chainweb.Time
+import Chainweb.Utils
+import Chainweb.Utils.Rule
+import Chainweb.Version
+
+import qualified Chainweb.BlockHeader.Genesis.FastDevelopment0Payload as FDN0
+import qualified Chainweb.BlockHeader.Genesis.FastDevelopment1to19Payload as FDNN
+
+pattern FastDevelopment :: ChainwebVersion
+pattern FastDevelopment <- ((== fastDevnet) -> True) where
+    FastDevelopment = fastDevnet
+
+fastDevnet :: ChainwebVersion
+fastDevnet = ChainwebVersion
+    { _versionCode = ChainwebVersionCode 0x00000002
+    , _versionName = ChainwebVersionName "fast-development"
+    , _versionForks = tabulateHashMap $ \_ -> AllChains ForkAtGenesis
+    , _versionUpgrades = AllChains mempty
+    , _versionGraphs = End twentyChainGraph
+    , _versionBlockRate = BlockRate 30_000_000
```

Comment:
`BlockDelay`?

Reply:
What does this comment mean?

Reply:
Oh, it's for considering renaming `BlockRate` to `BlockDelay` because that's what it really is.